app.js

```diff
@@ -73,13 +82,9 @@ function resolveFilePath(filePath) {
       return filePath;
     }
   }
-  ['.', __dirname].forEach(function (base) {
-    possiblePath = path.resolve(base, filePath);
-    if (fs.existsSync(possiblePath)) {
-      return possiblePath;
```

Comment:
Returning in a `forEach` doesn't actually do anything, so this didn't actually work.